scripts/avocado_featurize_submit

```diff
@@ -57,16 +76,21 @@ if __name__ == "__main__":
         default=None,
         help='Working directory. Default is the current directory.'
     )
+    parser.add_argument(
+        '--schedular',
+        default=None,
```

Comment:
Let's set the default to "sge" here rather than None.

Reply:
Resolved with [`abf05c7`](https://github.com/kboone/avocado/pull/6/commits/abf05c7388fab0888bfdd36fa7fade5d107156c7)